build/org.eclipse.pde.build.tests/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.pde</groupId>
+		<artifactId>eclipse.pde.build</artifactId>
+		<version>4.31.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>org.eclipse.pde.build.tests</artifactId>
+	<version>1.4.300-SNAPSHOT</version>
+	<packaging>eclipse-test-plugin</packaging>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>target-platform-configuration</artifactId>
+				<configuration>
+					<dependency-resolution>
+						<extraRequirements>
+							<requirement>
+								<!-- The product -->
+								<type>p2-installable-unit</type>
+								<id>org.eclipse.sdk.ide</id>
+								<versionRange>0.0.0</versionRange>
+							</requirement>
+						</extraRequirements>
+					</dependency-resolution>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-surefire-plugin</artifactId>
+				<configuration>
+					<product>org.eclipse.sdk.ide</product>
+					<testRuntime>p2Installed</testRuntime>
+					<testClass>org.eclipse.pde.build.tests.PDEBuildTestSuite</testClass>
+					<systemProperties>
+						<pde.build.includeP2>false</pde.build.includeP2>
+					</systemProperties>
+				</configuration>
+				<executions>
+					<execution>
+						<id>p2-tests</id>
+						<goals><goal>test</goal></goals>
+						<configuration>
+							<testClass>org.eclipse.pde.build.tests.P2TestSuite</testClass>
+							<systemProperties/>
```

Comment:
```suggestion
```
Is this necessary? The effective pom view says this element is not configured in a parent pom.

Reply:
This is to clear the inherited properties from the default execution/plugin configuration, one might can configure the default execution but lets dont complicate it too much as it already is :-)

Reply:
> but lets dont complicate it too much as it already is :-)

That's fine :)
But depending on where the to clear property is defined and how this might not be sufficient and something like https://stackoverflow.com/a/33252763/14542697 needs to be applied.
I have not further checked where this is coming from exactly, but I'm sure you are aware of all that.
Just wanted to make sure, it is not just forgotten.